files/models.py

```
@@ -1355,23 +1356,68 @@ def get_absolute_url(self):
-
+            temp_vtt = os.path.join(tmpdirname, "converted.vtt")
+            
+            cmd = [
+                settings.FFMPEG_COMMAND,  # Already configured in CinemataCMS
+                "-i", input_path,
+                "-c:s", "webvtt",
+                temp_vtt
+            ]
+            
+            try:
+                result = helpers.run_command(cmd)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)